R/autoplot.pop_data.R

```diff
@@ -6,68 +6,109 @@
 #' @param log whether to show antibody responses on logarithmic scale
 #' @param strata the name of a variable in `pop_data` to stratify by (or `NULL` for no stratification)
 #' @param ... unused
+#' @param type an option to choose type of chart, defaults to density chart
```

Comment:
- [ ] Please list available types in the documentation

Reply:
The list of types has been added.